warehouse/templates/pages/help.html

```diff
@@ -12,3 +12,71 @@
  # limitations under the License.
 -#}
 {% extends "base.html" %}
+
+{% block content %}
+  <section class="horizontal-section">
+    <div class="narrow-container">
+      <h2>Common Questions</h2>
+
+      <h3 id="installing" class="common-question">What is a package and how do I install one?</h3>
+
+      <p>A 'package' in the context of PyPI is a bundle of software to be installed.</p>
+      <p>Packages on PyPI are developed and shared by other members of the Python community so that you can use them in your own projects.</p>
+      <p>To learn how to install packages, visit the <a href="python-packaging-user-guide.readthedocs.io/en/latest/installing/">installation tutorial</a> on the <a href="python-packaging-user-guide.readthedocs.io/">Python Packaging User Guide</a>.</p>
```

Comment:
These URLs should probably point to ``packaging.python.org`` not ``python-packaging-user-guide.readthedocs.io`` I think? It's the same content, just a more official URL.